gammapy/makers/background/phase.py

```diff
@@ -108,25 +105,25 @@ def run(self, dataset, observation):
 
         acceptance_off = RegionNDMap.from_geom(geom=dataset.counts.geom)
         acceptance_off.data = np.sum([_[1] - _[0] for _ in self.off_phase])
-
+        
```

Comment:
Some formatting is needed. 
You can run black on each of the modified scripts or install a pre-commit hook as explained here : 

https://docs.gammapy.org/1.0/development/intro.html#:~:text=We%20adhere%20to,pre%2Dcommit%20install

Reply:
I am not sure to understand what you mean here

Reply:
The whitespace have to be removed to follow the codestyle convention.
To have this done automatically on before commit install a pre-commit hook in you gammapy folder :
`pip install pre-commit`
`pre-commit install`
here the commit is already done so I think that you need to apply the formatter directly
`pip install black`
`black gammapy/makers/background/phase.py`

Reply:
Okay I see thanks

Reply:
I just did it but you know for next time ;)